com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/AndroidBundler.java

```diff
@@ -109,27 +113,66 @@ private static Result exec(String... args) throws IOException {
         return exec(Arrays.asList(args));
     }
 
+
+    /**
+    * Get keystore. If none is provided a debug keystore will be generated in the project root and used
+    * when bundling.
+    * https://stackoverflow.com/a/4055893/1266551
+    */
+    private static String getKeystore(Project project) throws IOException {
+        String keystore = project.option("keystore", "");
+        if (keystore.length() == 0) {
+            keystore = "debug.keystore";
+            String keystoreAlias = "androiddebugkey";
+            String keystorePassword = "android";
+            String keystorePasswordFile = "debug.keystore.pass.txt";
+            if (!new File(keystore).exists()) {
+                Result r = exec("keytool",
```

Comment:
Is this tool available on Linux/Windows?

Reply:
Added method to use System.getProperty("java.home") or System.getenv("JAVA_HOME") when running keytool.